### ISSUE TITLE
When using MarkdownIt.parse() the resulting token markup is incorrect

### ISSUE THREAD

**Issue by lbevilacqua** [Moxio/markdown-it-fancy-lists#3]
The marker character position is calculated from the beginning of the state, instead from the beginning of the marker position.

To reproduce:
```markdown
### title

a. first item
#. second item
```

```javascript
const m = require('markdown-it'), mfl = require('markdown-it-fancy-lists');
const p = new m({ typographer: true }).use(mfl.markdownItFancyListPlugin);
p.parse(`
### title

a. first item
#. second item
`);
```
the detected markup for the list tokens is `#`

**Comment by lbevilacqua** [comment]
Proposed fix in PR #4.

**Comment by FritsvanCampen** [comment]
Thanks for the PR. I'll have a look at it this or next week.

**Comment by FritsvanCampen** [comment]
On input 
```markdown
### title

a. first item
#. second item
```
The output is
```html
<h3>title</h3>
<ol type="a">
<li>first item</li>
<li>second item</li>
</ol>
```
That seems correct to me. Your suggested change does not change the output.

The majority of this file has been based off of the original list parser: https://github.com/markdown-it/markdown-it/blob/master/lib/rules_block/list.js#L184

Can you supply a testcase that shows the problem?

**Comment by lbevilacqua** [comment]
The output of the render() method is correct (the HTML output).
The output of the parse() method is not.

I will provide a test case asap.

**Comment by FritsvanCampen** [comment]
Ah, I think I understand what you mean. The value of `posAfterMarker` is not calculated correctly. I think the correct value is `marker.posAfterMarker` and should be used instead.

I'm working on a fix.

**Comment by FritsvanCampen** [comment]
Fixed in 1.1.1.
